frontend/src/components/SessionTile.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components/macro';
+
+export default function SessionTile({ session }) {
+    return (
+        <Wrapper>
```

Comment:
Think about a different layout that doesn't need the `<br />`, also there is some unecessary nesting I think